plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -169,36 +169,37 @@ func TestWinPerfcountersConfigGet4(t *testing.T) {
 
 	perfobjects[0] = PerfObject
 
-	m := Win_PerfCounters{PrintValid: false, TestName: "ConfigGet4", Object: perfobjects}
+	m := Win_PerfCounters{PrintValid: false, Object: perfobjects}
 
-	err := m.ParseConfig(&metrics)
+	err := m.ParseConfig()
 	require.NoError(t, err)
 
-	if len(metrics.items) == 2 {
+	var parsedItems = m.GetParsedItemsForTesting()
+
+	if len(parsedItems) == 2 {
 		require.NoError(t, nil)
-	} else if len(metrics.items) < 2 {
+	} else if len(parsedItems) < 2 {
 
-		var errorstring1 string = "Too few results returned from the query: " + string(len(metrics.items))
+		var errorstring1 string = "Too few results returned from the query: " + string(len(parsedItems))
 		err2 := errors.New(errorstring1)
 		require.NoError(t, err2)
-	} else if len(metrics.items) > 2 {
+	} else if len(parsedItems) > 2 {
 
-		var errorstring1 string = "Too many results returned from the query: " + string(len(metrics.items))
+		var errorstring1 string = "Too many results returned from the query: " + string(len(parsedItems))
 		err2 := errors.New(errorstring1)
 		require.NoError(t, err2)
 	}
 }
 
 func TestWinPerfcountersConfigGet5(t *testing.T) {
-	metrics := itemList{}
 
 	var instances = make([]string, 2)
 	var counters = make([]string, 2)
 	var perfobjects = make([]perfobject, 1)
 
 	objectname := "Processor Information"
 	instances[0] = "_Total"
-	instances[1] = "0"
+	instances[1] = "0,1"
```

Comment:
Later versions of windows have multi-core support.

![image](https://user-images.githubusercontent.com/12101120/28739397-bc6cf9a8-73c8-11e7-8c78-e47803d9d624.png)